steps/deploy/step.yaml

```diff
@@ -19,21 +19,13 @@ examples:
     image: relaysh/helm-step-deploy
     spec:
       credentials:
-        ca.pem:
-          $type: Secret
-          name: ca
-        key.pem:
-          $type: Secret
-          name: key
-        cert.pem:
-          $type: Secret
-          name: cert
+        ca.pem: !Secret ca
+        key.pem: !Secret key
+        cert.pem: !Secret cert
       values:
         api:
           image:
-            tag: 
-              $type: Secret
-              name: api.image.tag
+            tag: !Secret api.image.tag
           adminToken:
             secretRef:
               name: admin-token

```

Comment:
Not quite sure what to do with these. These are chart-specific values right?

Reply:
We should probably use a real chart that exists in the wild instead of a bastardized version of one of our internal charts. I'll see what I can find for you.

Reply:
Yes, this is a chart value actually similar to one of our internal charts. Everything under `values` is a hash of chart values.